### DISCUSSION TITLE
Highlight Badge Not Showing

### DISCUSSION THREAD

**Discussion by dev-aniketj** [community/community#44966] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Highlight badge not showing in my profile after active the GitHub Education Pack.

**Comment by jge162** [comment]
it takes up to 24 hours, when did you do it?

**Comment by jge162** [comment, marked as answer]
@dev-aniketj did you get an email confirming you were accepted also?

**Reply by dev-aniketj** [reply]
Yes, I accept the verification.

It shows yesterday on my profile, but not it is hidden.
If they is sometimes in setting to visible the **Highlight** please let me know.

**Reply by jge162** [reply]
@dev-aniketj No setting to hide/show. It will just pop up eventually. Mine took like 1-2 days. It will show up. Just gotta be patient.

**Reply by jge162** [reply]
Hope that answers ur question.

**Reply by jge162** [reply]
Just browsed your read me. Looks really nice. Great work.
